src/TropicalGeometry/linear_space.jl

```diff
@@ -283,14 +284,27 @@ function tropical_linear_space(A::MatElem, nu::Union{Nothing,TropicalSemiringMap
     return TropL
 end
 
+# Special implementation for trivial valuation over the rationals using Polymake.tropical.matroid_fan for better performance
+function tropical_linear_space(A::QQMatrix, nu::TropicalSemiringMap{QQField, Nothing, minOrMax}=tropical_semiring_map(QQ); weighted_polyhedral_complex_only::Bool=false) where {minOrMax<:Union{typeof(min),typeof(max)}}
+    # Compute Bergman fan in polymake and convert to polyhedral complex
+    Sigma = PolyhedralComplex{QQFieldElem}(Polymake.tropical.matroid_fan{convention(nu)}(A))
+    # Add the missing all ones vector to the lineality
+    Sigma = add_missing_lineality_from_polymake(Sigma)
+    multiplicities = ones(ZZRingElem, n_maximal_polyhedra(Sigma))
```

Comment:
Perhaps one can consider the following instead (to avoid the costly computation of `n_maximal_polyhedra(Sigma)`). With this, the whole computation for a random 5-by-20 matrix `A` goes down from 30 to 4 seconds. (I'm not sure why this info isn't automatically carried over when we broadcast to `PolyhedralComplex{QQFieldElem}`?)

```suggestion
    # Compute Bergman fan in polymake and convert to polyhedral complex
    B = Polymake.tropical.matroid_fan{convention(nu)}(A)
    Sigma = PolyhedralComplex{QQFieldElem}(B)
    # Add the missing all ones vector to the lineality
    Sigma = Oscar.add_missing_lineality_from_polymake(Sigma)
    multiplicities = ones(ZZRingElem, B.N_MAXIMAL_POLYTOPES)
```

Reply:
The extra time for `n_maximal_polyhedra` is probably redundancy elimination after adding the extra lineality. And by not doing it here it will just delay it to the first time the user triggers some other computation with the object.

If it is guaranteed that this lineality does not change the combinatorics (and is not implicit in the other lineality rows) then adding the internal `non_redundant` flag for the constructor in `add_missing_lineality_from_polymake` might be a better approach as polymake will then directly store the properties as irredundant description.

Reply:
Done, thanks for the tip!  The `non_redundant` flat will come very useful for me.